compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -2465,7 +2466,8 @@ public void visit(BLangStringTemplateLiteral stringTemplateLiteral) {
 
     @Override
     public void visit(BLangWorkerSend workerSendNode) {
-        workerSendNode.expr = rewriteExpr(workerSendNode.expr);
+        List<BLangExpression> list = Arrays.asList(rewriteExpr(workerSendNode.expr));
```

Comment:
Can we use Lists.of(rewriteExpr(workerSendNode.expr)) as we used in other places in Desugar class. So we can get rid of with import as well.